### PR TITLE
Refresh content of user documents pages

### PR DESCRIPTION
**Pull request by dms1981**
## A reference to the issue / Description of it

#7279 

## How does this PR fix the problem?

Refactors the relevant pages for brevity and conciseness, includes some additional details based on team miro board.

## How has this been tested?

Test through CI

## Deployment Plan / Instructions

Deploy through CI

## Checklist (check `x` in `[ ]` of list items)

- [ ] I have performed a self-review of my own code
- [ ] All checks have passed
- [ ] I have made corresponding changes to the documentation
- [ ] Plan and discussed how it should be deployed to PROD (If needed)

## Additional comments (if any)

{Please write here}

### CODE REVIEW

File: source/team/ways-of-working.html.md.erb [hunk 1]
```diff
@@ -16,101 +16,49 @@ review_in: 6 months
...
-
-### Tags and Releases
-
-* Using the [Semver](https://semver.org/) versioning strategy, prefixed with a `v` e.g. `v1.*.*` to `v2.0.0` would represent a backwards incompatible API change.
-* Providing a descriptive title for the release, so it's easily identifiable and understood.
-* Providing a high level summary of `What's New` in the release, so consumers can easily understand the changes that have been made - and potentially any changes they may need to make when incrementing to the new version number
-* Making use of GitHub automatically generated release notes. These provide in depth detail of the release.
-    * These can be heavily populated general maintenance PRs. It is worth refining the commit list down to commits that will impact consumers. There is a full changelog link provided for people who want the additional information.
-
-### Reviewing Contributor Changes to Maintained Modules
```
Comment (mikereiddigital) [inline source/team/ways-of-working.html.md.erb:95]: Is there not value in keeping this section in the doc?

Comment (dms1981) [reply source/team/ways-of-working.html.md.erb:95]: I think I broadly cover that in these lines:
```
- Follow Semver for versioning
    - Provide a high-level `What's New` in the version release notes
    - Make use of GitHub's automatically generated release notes feature
```
But I'll add a little more detail in here!

Comment (mikereiddigital) [reply source/team/ways-of-working.html.md.erb:95]: Personally I think lines 95 to 113 provide useful information that isn't covered by the replacement text. Why is that section being deleted?

Comment (dms1981) [reply source/team/ways-of-working.html.md.erb:95]: In the spirit of my review I chose to replace it with a more concise section, but as you've pointed out that section is useful to new starters - I've pushed a new commit that highlights it better.

**Review by mikereiddigital — APPROVED** [review]
lg2m

**Pull request merged**